sentry/src/main/java/io/sentry/SentryReplayOptions.java

```diff
@@ -39,10 +51,10 @@ public final class SentryReplayOptions {
   private boolean redactAllImages = true;
 
   /**
-   * Defines the quality of the session replay. Higher bit rates have better replay quality, but
-   * also affect the final payload size to transfer, defaults to 100kbps.
+   * Defines the quality of the session replay. The higher the quality, the more accurate the replay
+   * will be, but also more data to transfer and more CPU load, defaults to MEDIUM.
    */
-  private int bitRate = 100_000;
+  private SentryReplayQuality quality = SentryReplayQuality.MEDIUM;
```

Comment:
@brustolin we said in the [ticket](https://github.com/getsentry/team-replay/issues/444) that the default should be `medium`, but I see you [use](https://github.com/getsentry/sentry-cocoa/blob/cd76c54171981b2f53636e9ee379725c9a3188b5/Sources/Swift/Integrations/SessionReplay/SentryReplayOptions.swift#L67) `low` - do you wanna change that, or should we re-phrase the ticket?

Reply:
Lets use medium, no problem.